.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Next.js v14: Debug full stack",
+      "type": "node-terminal",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "command": "npm run dev",
+      "serverReadyAction": {
+        "pattern": "- Local:        (https?://.+)",
```

Comment:
Note that this differs from my previous setup for Next.js v13 projects which used the pattern:

`"pattern": "started server on .+, url: (https?://.+)"`